### DISCUSSION TITLE
Imported GPX file with multiple tracks - Is it possible to change appearance of individual tracks?

### DISCUSSION THREAD

**Discussion by tmusolf** [osmandapp/OsmAnd#17022] [Q&A]
When importing a GPX file with multiple tracks is it possible to use the appearance menu to change the attributes of a single track?

As soon as you enter the appearance menu all of the tracks in the GPX file have their color and line width attributes changed to the same values - whatever is specified in the appearance menu.

It seems like it should be possible to edit and specify the following attributes for each individual track in a GPX file instead of all tracks in the file being treated the same:

- color
- line width
- split_interval / split_type
- show_arrows
- show_start_finish
- description

In addition, OSMAnd ignores the <desc> tag for an individual track.  It only recognizes a single <desc> tag for the entire GPX file contained in the <metadata> section.

It is often easier to manage a group of tracks and waypoints as a single imported GPX file instead of having the tracks imported as separate GPX files and having the waypoints spread around between the track files.  For example, a single GPX file having all the trails of a national park.  There are many shared waypoints between the tracks, you want to be able to quickly turn on/off the visibility of the entire GPX file, but yet have each track retain it's own properties.

**Comment by vshcherb** [comment]
For now it's only supported that you import and split these tracks into separate files and manage separately then
